connection/TypeDBTransactionImpl.java

```diff
@@ -81,78 +64,60 @@ public TypeDBOptions options() {
 
     @Override
     public boolean isOpen() {
-        return bidirectionalStream.isOpen();
+        if (!nativeObject.isOwned()) return false;
+        else return transaction_is_open(nativeObject);
     }
 
     @Override
     public ConceptManager concepts() {
-        return conceptMgr;
+        return new ConceptManagerImpl(nativeObject);
```

Comment:
Does it make sense to return a _new_ concept manager each time or should we should we cache and return the same one all the time?

Reply:
It's a simple projection, which shouldn't be expensive.

Reply:
Actually scratch that, this was needed before `nativeObject` became `final`, we can cache now.